openedx/core/djangoapps/notifications/email/utils.py

```diff
@@ -171,13 +171,10 @@ def get_start_end_date(cadence_type):
     """
     if cadence_type not in [EmailCadence.DAILY, EmailCadence.WEEKLY]:
         raise ValueError('Invalid cadence_type')
-    date_today = datetime.datetime.now()
-    yesterday = date_today - datetime.timedelta(days=1)
-    end_date = datetime.datetime.combine(yesterday, datetime.time.max)
-    start_date = end_date
+    end_date = datetime.datetime.now()
+    start_date = end_date - datetime.timedelta(days=1, minutes=15)
```

Comment:
I think making this 30 mins would be better

Reply:
I think 15 mins will be enough time. Normally, task status is changes from queued to in-progress in less than 15 minutes.